Formula/cabal-install.rb

```diff
@@ -12,7 +12,7 @@ class CabalInstall < Formula
 
   depends_on "ghc"
 
-  fails_with :clang if MacOS.version < :lion # Same as ghc.rb
+  fails_with :clang if MacOS.version <= :lion # Same as ghc.rb
```

Comment:
No need for updating `ghc.rb`. The test there is already `if MacOS.version <= :lion`.